src/clj/rems/css/styles.clj

```diff
@@ -641,9 +647,27 @@
    ;; These must be last as the parsing fails when the first non-standard element is met
    (generate-form-placeholder-styles)))
 
-(defn screen []
+(defn- render-css-file [content]
+  (let [dir-name "env/dev/resources/public/css"
+        file-name (str dir-name "/screen.css")
+        dir (java.io.File. dir-name)]
+    (log/info "Rendering CSS to file" (str (System/getProperty "user.dir") "/" file-name))
+    (when-not (.exists dir)
+      (.mkdirs dir))
+    (spit file-name content)))
+
+(defn screen-css []
   (g/css {:pretty-print? false} (remove-nil-vals (build-screen))))
 
-(deftest test-screen
+(deftest test-screen-css
   (binding [context/*lang* :fi]
-    (is (string? (rems.css.styles/screen)))))
+    (is (string? (screen-css)))))
+
+;; for development use and Figwheel updates
+(mount/defstate
+  rendered-css-file
+  :start
+  (when (env :render-css-file?)
+    (render-css-file
+     (binding [context/*lang* :en] ; default language to start Figwheel tracking
```

Comment:
so does the css file change language when `context/*lang*` changes?

Reply:
I think in the development flow it mostly goes so that you edit changes in `styles.clj` and reloading that file updates dynamically. I removed the rendering of the css file each time `screen.css` is requested because I thought it's not necessary but I could put it back. Now the language does not actually update.